.github/workflows/apply-konflux-manifests.yaml

```diff
@@ -20,9 +20,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Golang
-        uses: actions/setup-go@v5
-        with:
-          go-version-file: go.mod
+        uses: ./actions/setup-go
```

Comment:
is this path ok? in comparison to the rest

Reply:
As we checkout the repo into the root folder, I'd think so

Reply:
In the other ones, we checkout the hack repo to a custom path